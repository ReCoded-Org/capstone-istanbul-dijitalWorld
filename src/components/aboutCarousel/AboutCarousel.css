.allTeamsContent {
  margin-bottom: 6rem;
}

.desktopview .cardImg {
  width: 4.2rem;
  border-radius: 2.1rem;
}

.desktopview h4 {
  font-size: 1rem;
  font-weight: bold;
  color: #f67280;
}

.desktopview p {
  font-size: 0.9rem;
  color: #f67280;
}

.desktopview .teamCard {
  margin-bottom: 3rem;
}

.phoneView .cardImg {
  width: 3.5rem;
  border-radius: 1.7rem;
}

.phoneView h4 {
  font-size: 0.8rem;
  font-weight: bold;
  color: #f67280;
}

.phoneView p {
  font-size: 0.7rem;
  color: #f67280;
}

.tabletView .cardImg {
  width: 4rem;
  border-radius: 2rem;
}

.tabletView h4 {
  font-size: 0.8rem;
  font-weight: bold;
  color: #f67280;
}
.tabletView p {
  font-size: 0.7rem;
  color: #f67280;
}

@media only screen and (min-width: 776px) {
  .tabletView {
    display: none;
  }
  .phoneView {
    display: none;
  }
}

@media only screen and (max-width: 775px) and (min-width: 376px) {
  .desktopview {
    display: none;
  }
  .phoneView {
    display: none;
  }
}

@media only screen and (max-width: 375px) {
  .desktopview {
    display: none;
  }
  .tabletView {
    display: none;
  }
}
