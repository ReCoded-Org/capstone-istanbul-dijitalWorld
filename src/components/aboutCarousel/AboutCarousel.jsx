import React from 'react';
import './AboutCarousel.css';
import Carousel from 'react-bootstrap/Carousel';
import teamMember0 from './images/girl.svg';
import teamMember1 from './images/actress.svg';
import teamMember2 from './images/doctor.svg';
import teamMember3 from './images/businessMan.svg';
import teamMember4 from './images/callCenterAgent.svg';
import teamMember5 from './images/dentist.svg';
import { useTranslation } from 'react-i18next';
import { Col, Row, Container, Card } from 'react-bootstrap';

export default function AboutCarousel() {
  const { t } = useTranslation();
  return (
    <div allTeamsContent>
      <Container className="desktopView">
        <Row>
          <Col md>
            <Row className="teamCard">
              <Col sm="4">
                <img className="cardImg" src={teamMember0} alt="Team Member" />
              </Col>
              <Col md className="cardTexts">
                <h4>{t('about.name.3')}</h4>
                <p>{t('about.title.3')}</p>
              </Col>
            </Row>
          </Col>
          <Col md>
            <Row className="teamCard">
              <Col sm="4">
                <img className="cardImg" src={teamMember1} alt="Team Member" />
              </Col>
              <Col md className="cardTexts">
                <h4>{t('about.name.4')}</h4>
                <p>{t('about.title.4')}</p>
              </Col>
            </Row>
          </Col>
          <Col md>
            <Row className="teamCard">
              <Col sm="4">
                <img className="cardImg" src={teamMember2} alt="Team Member" />
              </Col>
              <Col md className="cardTexts">
                <h4>{t('about.name.5')}</h4>
                <p>{t('about.title.5')}</p>
              </Col>
            </Row>
          </Col>
        </Row>
        <Row>
          <Col md>
            <Row className="teamCard">
              <Col sm="4">
                <img className="cardImg" src={teamMember3} alt="Team Member" />
              </Col>
              <Col md className="cardTexts">
                <h4>{t('about.name.0')}</h4>
                <p>{t('about.title.0')}</p>
              </Col>
            </Row>
          </Col>
          <Col md>
            <Row className="teamCard">
              <Col sm="4">
                <img className="cardImg" src={teamMember4} alt="Team Member" />
              </Col>
              <Col md className="cardTexts">
                <h4>{t('about.name.1')}</h4>
                <p>{t('about.title.1')}</p>
              </Col>
            </Row>
          </Col>
          <Col md>
            <Row className="teamCard">
              <Col sm="4">
                <img className="cardImg" src={teamMember5} alt="Team Member" />
              </Col>
              <Col md className="cardTexts">
                <h4>{t('about.name.2')}</h4>
                <p>{t('about.title.2')}</p>
              </Col>
            </Row>
          </Col>
        </Row>
      </Container>
      <Container className="tabletView">
        <Carousel>
          <Carousel.Item>
            <Row>
              <Col>
                <Card border="white" style={{ width: '12rem' }} className="memberCard">
                  <Card.Body className="text-center">
                    <img className="cardImg" src={teamMember0} alt="Team Member" />
                    <h4>{t('about.name.3')}</h4>
                    <p>{t('about.title.3')}</p>
                  </Card.Body>
                </Card>
              </Col>
              <Col>
                <Card border="white" style={{ width: '12rem' }} className="memberCard">
                  <Card.Body className="text-center">
                    <img className="cardImg" src={teamMember1} alt="Team Member" />
                    <h4>{t('about.name.4')}</h4>
                    <p>{t('about.title.4')}</p>
                  </Card.Body>
                </Card>
              </Col>
              <Col>
                <Card border="white" style={{ width: '12rem' }} className="memberCard">
                  <Card.Body className="text-center">
                    <img className="cardImg" src={teamMember2} alt="Team Member" />
                    <h4>{t('about.name.5')}</h4>
                    <p>{t('about.title.5')}</p>
                  </Card.Body>
                </Card>
              </Col>
            </Row>
          </Carousel.Item>
          <Carousel.Item>
            <Row>
              <Col>
                <Card border="white" style={{ width: '12rem' }} className="memberCard">
                  <Card.Body className="text-center">
                    <img className="cardImg" src={teamMember3} alt="Team Member" />
                    <h4>{t('about.name.0')}</h4>
                    <p>{t('about.title.0')}</p>
                  </Card.Body>
                </Card>
              </Col>
              <Col>
                <Card border="white" style={{ width: '12rem' }} className="memberCard">
                  <Card.Body className="text-center">
                    <img className="cardImg" src={teamMember4} alt="Team Member" />
                    <h4>{t('about.name.1')}</h4>
                    <p>{t('about.title.1')}</p>
                  </Card.Body>
                </Card>
              </Col>
              <Col>
                <Card border="white" style={{ width: '12rem' }} className="memberCard">
                  <Card.Body className="text-center">
                    <Card.Img className="cardImg" src={teamMember5} alt="Team Member"></Card.Img>
                    <h4>{t('about.name.2')}</h4>
                    <p>{t('about.title.2')}</p>
                  </Card.Body>
                </Card>
              </Col>
            </Row>
          </Carousel.Item>
        </Carousel>
      </Container>
      <Container className="phoneView">
        <Carousel>
          <Carousel.Item className="carouselCardPhoneView">
            <Card border="white" style={{ width: '50%' }} className="memberCard">
              <Card.Body className="text-center">
                <img className="cardImg" src={teamMember0} alt="Team Member" />
                <h4>{t('about.name.3')}</h4>
                <p>{t('about.title.3')}</p>
              </Card.Body>
            </Card>
          </Carousel.Item>
          <Carousel.Item className="carouselCardPhoneView">
            <Card border="white" style={{ width: '50%' }} className="memberCard">
              <Card.Body className="text-center">
                <img className="cardImg" src={teamMember1} alt="Team Member" />
                <h4>{t('about.name.4')}</h4>
                <p>{t('about.title.4')}</p>
              </Card.Body>
            </Card>
          </Carousel.Item>
          <Carousel.Item className="carouselCardPhoneView">
            <Card border="white" style={{ width: '50%' }} className="memberCard">
              <Card.Body className="text-center">
                <img className="cardImg" src={teamMember2} alt="Team Member" />
                <h4>{t('about.name.5')}</h4>
                <p>{t('about.title.5')}</p>
              </Card.Body>
            </Card>
          </Carousel.Item>
          <Carousel.Item className="carouselCardPhoneView">
            <Card border="white" style={{ width: '50%' }} className="memberCard">
              <Card.Body className="text-center">
                <img className="cardImg" src={teamMember3} alt="Team Member" />
                <h4>{t('about.name.0')}</h4>
                <p>{t('about.title.0')}</p>
              </Card.Body>
            </Card>
          </Carousel.Item>
          <Carousel.Item className="carouselCardPhoneView">
            <Card border="white" style={{ width: '50%' }} className="memberCard">
              <Card.Body className="text-center">
                <img className="cardImg" src={teamMember4} alt="Team Member" />
                <h4>{t('about.name.1')}</h4>
                <p>{t('about.title.1')}</p>
              </Card.Body>
            </Card>
          </Carousel.Item>
          <Carousel.Item className="carouselCardPhoneView">
            <Card border="white" style={{ width: '50%' }} className="memberCard">
              <Card.Body className="text-center">
                <img className="cardImg" src={teamMember5} alt="Team Member" />
                <h4>{t('about.name.2')}</h4>
                <p>{t('about.title.2')}</p>
              </Card.Body>
            </Card>
          </Carousel.Item>
        </Carousel>
      </Container>
    </div>
  );
}
