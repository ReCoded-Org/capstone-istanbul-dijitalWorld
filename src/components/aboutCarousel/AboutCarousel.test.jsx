import React from 'react';
import AboutCarousel from './AboutCarousel';
import renderer from 'react-test-renderer';

it('about carousel renders correctly', () => {
  const tree = renderer.create(<AboutCarousel />).toJSON();
  expect(tree).toMatchSnapshot();
});
