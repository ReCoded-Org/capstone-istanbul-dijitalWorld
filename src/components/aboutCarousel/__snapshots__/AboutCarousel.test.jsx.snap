// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about carousel renders correctly 1`] = `
<div
  allTeamsContent={true}
>
  <div
    className="desktopview container"
  >
    <div
      className="row"
    >
      <div
        className="col-md"
      >
        <div
          className="teamCard row"
        >
          <div
            className="col-sm-4"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="girl.svg"
            />
          </div>
          <div
            className="cardTexts col-md"
          >
            <h4>
              about.name.3
            </h4>
            <p>
              about.title.3
            </p>
          </div>
        </div>
      </div>
      <div
        className="col-md"
      >
        <div
          className="teamCard row"
        >
          <div
            className="col-sm-4"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="actress.svg"
            />
          </div>
          <div
            className="cardTexts col-md"
          >
            <h4>
              about.name.4
            </h4>
            <p>
              about.title.4
            </p>
          </div>
        </div>
      </div>
      <div
        className="col-md"
      >
        <div
          className="teamCard row"
        >
          <div
            className="col-sm-4"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="doctor.svg"
            />
          </div>
          <div
            className="cardTexts col-md"
          >
            <h4>
              about.name.5
            </h4>
            <p>
              about.title.5
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md"
      >
        <div
          className="teamCard row"
        >
          <div
            className="col-sm-4"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="businessMan.svg"
            />
          </div>
          <div
            className="cardTexts col-md"
          >
            <h4>
              about.name.0
            </h4>
            <p>
              about.title.0
            </p>
          </div>
        </div>
      </div>
      <div
        className="col-md"
      >
        <div
          className="teamCard row"
        >
          <div
            className="col-sm-4"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="callCenterAgent.svg"
            />
          </div>
          <div
            className="cardTexts col-md"
          >
            <h4>
              about.name.1
            </h4>
            <p>
              about.title.1
            </p>
          </div>
        </div>
      </div>
      <div
        className="col-md"
      >
        <div
          className="teamCard row"
        >
          <div
            className="col-sm-4"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="dentist.svg"
            />
          </div>
          <div
            className="cardTexts col-md"
          >
            <h4>
              about.name.2
            </h4>
            <p>
              about.title.2
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="tabletView container"
  >
    <div
      className="carousel slide"
      onKeyDown={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
    >
      <ol
        className="carousel-indicators"
      >
        <li
          className="active"
          onClick={[Function]}
        />
        <li
          onClick={[Function]}
        />
      </ol>
      <div
        className="carousel-inner"
      >
        <div
          className="active carousel-item"
        >
          <div
            className="row"
          >
            <div
              className="col"
            >
              <div
                className="teamCard row"
              >
                <div
                  className="col"
                >
                  <img
                    alt="Team Member"
                    className="cardImg"
                    src="girl.svg"
                  />
                </div>
                <div
                  className="cardTexts col"
                >
                  <h4>
                    about.name.5
                  </h4>
                  <p>
                    about.title.5
                  </p>
                </div>
              </div>
            </div>
            <div
              className="col"
            >
              <div
                className="teamCard row"
              >
                <div
                  className="col"
                >
                  <img
                    alt="Team Member"
                    className="cardImg"
                    src="actress.svg"
                  />
                </div>
                <div
                  className="cardTexts col"
                >
                  <h4>
                    about.name.4
                  </h4>
                  <p>
                    about.title.4
                  </p>
                </div>
              </div>
            </div>
            <div
              className="col"
            >
              <div
                className="teamCard row"
              >
                <div
                  className="col"
                >
                  <img
                    alt="Team Member"
                    className="cardImg"
                    src="doctor.svg"
                  />
                </div>
                <div
                  className="cardTexts col"
                >
                  <h4>
                    about.name.5
                  </h4>
                  <p>
                    about.title.5
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <div
            className="row"
          >
            <div
              className="col"
            >
              <div
                className="teamCard row"
              >
                <div
                  className="col"
                >
                  <img
                    alt="Team Member"
                    className="cardImg"
                    src="businessMan.svg"
                  />
                </div>
                <div
                  className="cardTexts col"
                >
                  <h4>
                    about.name.0
                  </h4>
                  <p>
                    about.title.0
                  </p>
                </div>
              </div>
            </div>
            <div
              className="col"
            >
              <div
                className="teamCard row"
              >
                <div
                  className="col"
                >
                  <img
                    alt="Team Member"
                    className="cardImg"
                    src="callCenterAgent.svg"
                  />
                </div>
                <div
                  className="cardTexts col"
                >
                  <h4>
                    about.name.1
                  </h4>
                  <p>
                    about.title.1
                  </p>
                </div>
              </div>
            </div>
            <div
              className="col"
            >
              <div
                className="teamCard row"
              >
                <div
                  className="col"
                >
                  <img
                    alt="Team Member"
                    className="cardImg"
                    src="dentist.svg"
                  />
                </div>
                <div
                  className="cardTexts col"
                >
                  <h4>
                    about.name.2
                  </h4>
                  <p>
                    about.title.2
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <a
        className="carousel-control-prev"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-prev-icon"
        />
        <span
          className="sr-only"
        >
          Previous
        </span>
      </a>
      <a
        className="carousel-control-next"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-next-icon"
        />
        <span
          className="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
  <div
    className="phoneView container"
  >
    <div
      className="carousel slide"
      onKeyDown={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
    >
      <ol
        className="carousel-indicators"
      >
        <li
          className="active"
          onClick={[Function]}
        />
        <li
          onClick={[Function]}
        />
        <li
          onClick={[Function]}
        />
        <li
          onClick={[Function]}
        />
        <li
          onClick={[Function]}
        />
        <li
          onClick={[Function]}
        />
      </ol>
      <div
        className="carousel-inner"
      >
        <div
          className="active carousel-item"
        >
          <div
            className="teamCard"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="girl.svg"
            />
            <h4>
              about.name.3
            </h4>
            <p>
              about.title.3
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <div
            className="teamCard"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="actress.svg"
            />
            <h4>
              about.name.4
            </h4>
            <p>
              about.title.4
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <div
            className="teamCard"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="doctor.svg"
            />
            <h4>
              about.name.5
            </h4>
            <p>
              about.title.5
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <div
            className="teamCard"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="businessMan.svg"
            />
            <h4>
              about.name.0
            </h4>
            <p>
              about.title.0
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <div
            className="teamCard"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="callCenterAgent.svg"
            />
            <h4>
              about.name.1
            </h4>
            <p>
              about.title.1
            </p>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <div
            className="teamCard"
          >
            <img
              alt="Team Member"
              className="cardImg"
              src="dentist.svg"
            />
            <h4>
              about.name.2
            </h4>
            <p>
              about.title.2
            </p>
          </div>
        </div>
      </div>
      <a
        className="carousel-control-prev"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-prev-icon"
        />
        <span
          className="sr-only"
        >
          Previous
        </span>
      </a>
      <a
        className="carousel-control-next"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-next-icon"
        />
        <span
          className="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</div>
`;
